classifier.py

```
@@ -1,552 +1,552 @@
+        return self.model
+
+    def train_nb(self, X_train, y_train):
+        """训练朴素贝叶斯分类器"""
+        print(f"\n训练朴素贝叶斯分类器...")
+
+        self.model = GaussianNB()
+        self.model.fit(X_train, y_train)
+        print("朴素贝叶斯训练完成")
+        return self.model
+
+    def train_lda(self, X_train, y_train, n_neighbors=5):
```

Ruff (0.14.1)

Unused method argument: `n_neighbors`

(ARG002)